source/MaterialXGraphEditor/Main.cpp

```diff
@@ -100,6 +102,10 @@ int main(int argc, char* const argv[])
         {
             parseToken(nextToken, "string", captureFilename);
         }
+        else if (token == "--scaleFactor")
+        {
+            parseToken(nextToken, "float", scaleFactor);
```

Comment:
The only firewall test I suggest is to make sure negative numbers aren't allowed.